components/prism-birb.js

```diff
@@ -0,0 +1,25 @@
+Prism.languages.birb = Prism.languages.extend('clike', {
+	'class-name': [
+		/\b[A-Z](?:\w*[a-zA-Z]\w*)?\b/,
+    	/\b[A-Z]\w*(?=\s+\w+\s*[;,=())])/
+    ],
+	'keyword': /\b(?:assert|bool|break|case|class|const|default|double|else|enum|final|follows|for|grab|if|int|nest|next|new|noSeeb|return|static|switch|throw|var|void|while)\b/,
```

Comment:
`bool`, `double`, and `int` are keywords? They're not in the [keyword list](https://birbolang.web.app/docs/#keywords).